exporter/awsxrayexporter/translator/cause.go

```diff
@@ -346,3 +348,72 @@ func fillPythonStacktrace(stacktrace string, exceptions []awsxray.Exception) []a
 
 	return exceptions
 }
+
+func fillJavaScriptStacktrace(stacktrace string, exceptions []awsxray.Exception) []awsxray.Exception {
+	r := textproto.NewReader(bufio.NewReader(strings.NewReader(stacktrace)))
+
+	// Skip first line containing top level exception / message
+	r.ReadLine()
+	exception := &exceptions[0]
+	var line string
+	line, err := r.ReadLine()
+	if err != nil {
+		return exceptions
+	}
+
+	exception.Stack = make([]awsxray.StackFrame, 0)
+	for {
+		if strings.HasPrefix(line, "    at ") {
+			parenIdx := strings.IndexByte(line, '(')
+			atIdx := strings.Index(line, "at ")
```

Comment:
Don't need to search now since we have checked the prefix, we know the index.

Reply:
fixed